app/views/publishers/vacancies/build/about_the_role.html.slim

```diff
@@ -7,6 +7,18 @@
 
       = vacancy_form_page_heading(vacancy, step_process, back_path: back_path, fieldset: false)
 
+      - if vacancy.organisation&.description.present?
+        p.govuk-body = t("helpers.label.publishers_job_listing_about_the_role_form.school_profile.description")
+        = govuk_details(summary_text: t("helpers.label.publishers_job_listing_about_the_role_form.school_profile.description_summary_text")) do
```

Comment:
Feels a little weird for these translations to be found at helpers.labels in my opinion as they're not really form labels. Perhaps they could be put in `publishers.yml` somewhere instead?